pkg/config/config.go

```diff
@@ -179,5 +179,8 @@ func (rule *Rule) mapToConfig(result map[string]any) error {
 
 	rule.Extra = result
 
+	delete(rule.Extra, "level")
+	delete(rule.Extra, "ignore")
```

Comment:
💭 does it alter `result`? Would we care?

Reply:
It does, and it's a good point! But at least for now it's an internal method which is only called after a string has been unmarshalled into a map. I'll add a comment to make sure we're aware of that if its use case is extended later.